test/framework/resources/templates/block-loki-validatingwebhookconfiguration.yaml.tpl

```diff
@@ -0,0 +1,43 @@
+---
```

Comment:
Why don't you simply delete the HVPA for the loki during the integration test?

Reply:
Because I think that if there is a shoot reconciliation during the integration test, The hvpa will be deployed again, am I wrong?